db/migrate/20190102161442_add_taggings_for_all_taggable_tables.rb

```diff
@@ -0,0 +1,41 @@
+class AddTaggingsForAllTaggableTables < ActiveRecord::Migration[5.1]
+  def change
+    create_table "tags", id: :serial, force: :cascade do |t|
+      t.references :tenant, :type => :bigint, :index => false, :null => false, :foreign_key => {:on_delete => :cascade}
+
+      t.string "name", :null => false
+      t.text "description"
+
+      t.boolean "active", :default => true, :null => false
+      t.boolean "multiple", :default => false, :null => false
+
+      t.index ["tenant_id", "name"], :unique => true
+    end
+
+    create_table "vms_tags", id: :serial, force: :cascade do |t|
```

Comment:
`"vm_tags"` would make all the associations easier, with `VmsTags` we're going to have to define the class name on all the assocs

Reply:
you're probably right. I think I'll be able to do our own act_as_a_taggable, what will just generate the right relations names. 

If we'll use ```vm_tags```, we'll need to call singularize on when generationg the inventory collectio name.